### ISSUE TITLE
[Bug]: [metrics] Less accurate quernode monitoring metrics

### ISSUE THREAD

**Issue by ThreadDao** [milvus-io/milvus#22302]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: master-20230218-57f8de95
- Deployment mode(standalone or cluster): cluster
- MQ type(rocksmq, pulsar or kafka):  pulsar  
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

1. `DML Virtual Channel`
  - Sometimes it's negative and sometimes it's 0
![image](https://user-images.githubusercontent.com/27288593/220086242-fbde4ad1-56d3-4e0e-bd1c-1757a246a648.png)

2. `Queryable Entity Num`
- There are 3 querynodes, only 2 querynode are shown in the metrics. Expected to show 0 if the querynode is not loaded
![image](https://user-images.githubusercontent.com/27288593/220088076-37d40805-1a59-4008-a498-22cbfabad596.png)

- The unit is k, which cannot be displayed accurately
![image](https://user-images.githubusercontent.com/27288593/220088763-e00b4f68-1f24-4c09-9e04-e22ea8c88c18.png)

![image](https://user-images.githubusercontent.com/27288593/220091055-2c056c09-933e-4422-a5a3-0d10b3b90baf.png)

**Comment by xige-16** [comment]
tht bug has been fix on branch 2.2.0,

**Comment by xige-16** [comment]
TODO :: pick the fix to master branch
